source/block-reconciler.ts

```diff
@@ -1,28 +1,25 @@
 import { Block } from "./models/block";
 import { BlockHistory } from "./models/block-history";
-import { List as ImmutableList } from "immutable";
 
-type GetBlockByHash<TBlock> = (hash: string) => Promise<TBlock|null>;
+type GetBlockByHash<TBlock> = (hash: string) => Promise<TBlock | null>;
```

Comment:
I'm fine with this.  My editor already tells me to do this, I just miss the notice sometimes.